integrations/lib/multiwoven/integrations/source/aws_athena/config/spec.json

```diff
@@ -0,0 +1,62 @@
+{
+  "documentation_url": "https://docs.multiwoven.com/integrations/sources/athena",
+  "stream_type": "dynamic",
+  "connector_query_type": "raw_sql",
+  "connection_specification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "AWS Athena",
+    "type": "object",
+    "required": ["region", "workgroup", "catalog", "database", "output_location"],
+    "properties": {
+      "access_key": {
+        "description": "The AWS Access Key ID to use for authentication.",
+        "examples": ["AWSATHENAEXAMPLE"],
+        "type": "string",
+        "title": "Personal Access Key",
+        "order": 0
+      },
+      "secret_access_key": {
+        "description": "The AWS Secret Access Key to use for authentication.",
+        "examples": ["AWS/ATHENA/EXAMPLEKEY"],
+        "type": "string",
+        "title": "Secret Access Key",
```

Comment:
@TivonB-AI2  add multiwoven_secret to true to mask this in the ui for credentails. refer other connectors

Reply:
Done